vowpalwabbit/cb_label_parser.h

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) by respective owners including Yahoo!, Microsoft, and
+// individual contributors. All rights reserved. Released under a BSD (revised)
+// license as described in the file LICENSE.
+
+#pragma once
+#include "cb.h"
+
+namespace CB
+{
+  template<typename LBL = CB::label>
```

Comment:
Nit: For a template argument this should probably be LabelT

Reply:
Done.